src/inputs/labeled-inputs.scss

```diff
@@ -87,17 +78,17 @@
     min-height: $iui-line-height;
   }
 
-  &.iui-disabled > label {
-    cursor: not-allowed;
-  }
-
   &.iui-inline {
     flex-direction: row;
     align-items: center;
 
     > .iui-label {
       flex-shrink: 0;
-      margin: 0 $iui-sm 0 0;
+      margin: 0 ($iui-m + 2) 0 0;
```

Comment:
It looks more centered with margin bottom 3px than not, to be honest :D

These +2 come from...?

Reply:
Is that the least margin we could have? Required is not that often compared how many places it influenced, and in general, I think endresult is too wide (looking at filters and other places, where there is no required...)

Reply:
what does margin-bottom have to do with anything?

the `+ 2` is just the value I found which aligned this label with the required one.

I could try making it even tighter, but the visuals were approved by Jon before he left.

Reply:
Margin-bottom is overriden with this rule, so just commented, what I was playnig around. Not sure, if that was influenced by your changes.
I just checked the images, thought that we expanded in width too much, compared, the content did not really change. Especially, when we talk a lot about condensed and stuff..
But I guess we can leave it, if thats ok for everyone.

Then +2 look kinda random.

Reply:
The margin-bottom was not influenced by my changes, we always override it with 0 for inline label.

Reply:
Reduced overall margin-right by 2px, so the `+ 2` is gone now.

The value is now 6px on required, and 16px on non-required.
![image](https://user-images.githubusercontent.com/9084735/131401912-b957c801-989b-43e5-af9e-83582753c881.png)
![image](https://user-images.githubusercontent.com/9084735/131401944-8f56cf75-98c0-43a6-a0c9-ce12e9936508.png)

I also tried changing the ::after `content` from `' *'` to `'*'` and adding margin-left: 2px, but I can't get the checkboxes to align so I didn't push.
![image](https://user-images.githubusercontent.com/9084735/131403342-0bef8dfc-71bb-4a2f-a7a4-f018f6575577.png)
![image](https://user-images.githubusercontent.com/9084735/131403316-2c702b52-70cd-4a47-b3f8-bc117d59df44.png)